code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -273,6 +275,7 @@
 	if(!isturf(holder))
 		return NO_REACTION
 	var/turf/open/location = holder
+	var/list/cached_results = air.reaction_results
```

Comment:
Caching this list in this instance is slower, because you need to both create a new variable and access the list
datum var access are slow, but if you're only doing them once there's no point caching it